eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -5,7 +5,9 @@
   <IgnorePatterns>
     <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
 
-    <!-- 
+    <Usage Id="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
@dotnet/source-build-internal how do I fix "new prebuilts detected" for Newtonsoft.Json? Is this fine?

Reply:
Thanks for asking @jjonescz.  Source build must build all dependent binaries from source.  External dependencies such as Newtonsoft.Json are contained in the https://github.com/dotnet/source-build-externals repo.  To fix this prebuilt, you simply need to add the appropriate version.details.xml dependency to this repo and add a darc subscription to keep it up to date.

```
    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="8.0.0-alpha.1.23312.1">
      <Uri>https://github.com/dotnet/source-build-externals</Uri>
      <Sha>50600f7fcd6e56a496233a859ed7d4d90c173b0d</Sha>
      <SourceBuild RepoName="source-build-externals" ManagedOnly="true" />
    </Dependency>
```

When razor CI runs the source-build leg, this package will be pulled in by the source-build infrastructure from source-build-externals and therefore will not be considered a prebuilt.

Reply:
That's what I originally tried, but it doesn't get rid of the error. I redid the change, so you can see - it fails currently in CI. Can the reason be that I'm using Newtonsoft.Json version 13.0.3 but there's 13.0.1 in source-build-externals (I think)? But using 13.0.1 explicitly would get NuGet complaining about package downgrade (since there's Newtonsoft.Json 13.0.3 flowing into other projects from Roslyn) - although I guess I could use 13.0.1 for this source-built project and keep 13.0.3 for others if that will solve the problem...